rotkehlchen/tests/unit/test_inquirer.py

```diff
@@ -521,7 +521,10 @@ def test_punishing_of_oracles_works(inquirer):
                 assert coingecko_mock.called is True
                 assert defillama_mock.called is True
 
+        # move the current time forward and check that coingecko is still penalized
+        with freeze_time(datetime.utcfromtimestamp(ts_now() + ORACLE_PENALTY_TS // 2)):
```

Comment:
There's no need to do an integer division here since `datetime.utcfromtimestamp()` accepts a float
```suggestion
        with freeze_time(datetime.utcfromtimestamp(ts_now() + ORACLE_PENALTY_TS / 2)):
```